src/http/RequestMethod.h

```diff
@@ -27,6 +27,11 @@ class HttpRequestMethod : public RefCountable
 {
 public:
     HttpRequestMethod() : theMethod(Http::METHOD_NONE), theImage() {}
+    HttpRequestMethod(const HttpRequestMethod & aMethod) {
+        theMethod = aMethod.theMethod;
+        theImage = aMethod.theImage;
+        // do not copy the count_ member inherited from RefCountable
+    }
```

Comment:
> I would prefer to tackle this as a separate PR immediately after this one

Then my earlier bug-preserving suggestion applies (provided there are no more new warnings like this one):

```suggestion
    // XXX: Lock descendants must not be copied like that!
    HttpRequestMethod(const HttpRequestMethod &) = default;
    HttpRequestMethod(const HttpRequestMethod &&) = delete;
```